app/src/main/java/com/karumi/kodein/sample/MainActivityController.kt

```diff
@@ -0,0 +1,5 @@
+package com.karumi.kodein.sample
+
+class MainActivityController(val nameDataSource: NameDataSource) {
```

Comment:
I guess we just want to show the usage of the dependency injector in the concrete scenario you want to, so I'd change the code to have just one dependency and not the controller and also the data source. This could simplify the example a lot. Just one class injected instead of two 😃  What do you think?

Reply:
sounds good, I'm going to change the name to the classes to activityScope and applictionScope or something like that. Thank you